http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriberSpec.scala

```diff
@@ -370,6 +370,45 @@ class LeoPubsubMessageSubscriberSpec
     res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
 
+  /**
+   * When createRuntime gets 409 or a cluster already exist error, we shouldn't attempt to update AsyncRuntimeFields.
+   * These fields should've been updated in a previous createRuntime request, and
+   * this test is to make sure we're not wiping out that info.
+   */
+  it should "handle CreateRuntimeMessage properly when google returns 409 or AlreadyExists error for dataproc" in isolatedDbTest {
+    val dataprocAlg = new BaseMockRuntimeAlgebra {
+      override def createRuntime(params: CreateRuntimeParams)(implicit
+        ev: Ask[IO, AppContext]
+      ): IO[Option[CreateGoogleRuntimeResponse]] = IO.pure(None)
+    }
+
+    val asyncFields = makeAsyncRuntimeFields(1)
+
+    val leoSubscriber = makeLeoSubscriber(dataprocRuntimeAlgebra = dataprocAlg)
+    val res =
+      for {
+        runtime <- IO(
+          makeCluster(1)
+            .copy(serviceAccount = serviceAccount, asyncRuntimeFields = None, status = RuntimeStatus.Creating)
```

Comment:
should the `asyncFields` here be `asyncFields`?

Reply:
Thanks, good catch :)